ddtrace/internal/telemetry/writer.py

```diff
@@ -264,18 +261,17 @@ def add_event(self, payload, payload_type):
             Payload types accepted by telemetry/proxy v2: app-started, app-closing, app-integrations-change
         """
         if self.enable():
-            event = {
-                "tracer_time": int(time.time()),
-                "runtime_id": get_runtime_id(),
-                "api_version": "v2",
-                "seq_id": next(self._sequence_payloads),
-                "debug": self._debug,
-                "application": get_application(config.SERVICE, config.VERSION, config.ENV),
-                "host": get_host_info(),
-                "payload": payload,
-                "request_type": payload_type,
-            }
-            self._events_queue.append(event)
+            with self._service_lock:
+                self._events_queue.append({"payload": payload, "request_type": payload_type})
+
+    def add_events(self, events):
+        # type: (List[Dict[str, Any]]) -> None
```

Comment:
While we're here can we fix the deprecation warnings `Use of type comments is deprecated; use type annotation instead`, e.g.:

```suggestion
    def add_events(self, events: List[Dict[str, Any]]) -> None:
```

Reply:
I opened up this PR to update the typing: https://github.com/DataDog/dd-trace-py/pull/14599